ChangeLog.d/mbedtls_ssl_cert_cb.txt

```diff
@@ -0,0 +1,4 @@
+Features
+   * Add server certificate selection callback near end of Client Hello
+   * Provide mechanism to reset handshake cert list
```

Comment:
Same: please mention this is done by calling `mbedtls_ssl_set_hs_own_cert()` with `own_cert == NULL`.

Reply:
:rocket: